spec/features/system/admin/user_management_spec.rb

```diff
@@ -81,7 +81,9 @@
         find('div[aria-label="Search"]').find('input').set(random_user.email)
         wait_for_field_debouncing # timeout for search debouncing
 
-        expect(page).to have_selector('div.user_name', text: random_user.name)
+        non_searched_user = users_to_search.reject { |user| user.id == random_user.id }.first
```

Comment:
Actually instead of adding extra logic here, can reference splitting the scenario into a separate one:
see `coursemology2/spec/features/system/admin/instance/user_management_spec.rb:89`

Can also update the `admin/instance/user_management_spec.rb` to check `.email` using `have_selector` instead of checking `.name` using `have_text` then.

Reply:
done